convention-develocity-gradle-plugin/README.md

```diff
@@ -1,8 +1,39 @@
 ## Convention Develocity Gradle Plugin
 
-This project demonstrates how a convention plugin can share the same Develocity build configuration across multiple projects. It 
-is intended to serve as a starting point for creating your own Gradle plugin that applies your specific Develocity configuration. Note
-the inline comments in the build and source code for things to adjust specifically to your needs.
+This project demonstrates how a convention plugin can share the same Develocity build configuration across multiple projects.
+It is intended to serve as a starting point for creating your own Gradle plugin that applies your specific Develocity configuration.
+Note the inline comments in the build and source code for things to adjust specifically to your needs.
+
+### Applying the plugin to your build
+
+#### Using a dynamic version
+
+For builds using Gradle 6.9 and later, it's recommended to apply the convention plugin using a [dynamic version](https://docs.gradle.org/current/userguide/dynamic_versions.html).
+This approach ensures that the latest version of the plugin is resolved, up to the specified maximum version.
+For example, the following resolves the latest version of the plugin up to, but excluding, version `2.0.0`:
+
+```groovy
+plugins {
+    id 'com.myorg.convention-develocity-gradle-5-or-newer' version '(,2.0.0)'
```

Comment:
How come you are not using the `1.+` syntax?

Reply:
I discussed this in a call with @etiennestuder. For other reviewers, I had done this for consistency with Maven. However, we decided it's okay to deviate and use the `+` syntax since it's more idiomatic and arguably more intuitive.